src/test/ui/suggestions/match-prev-arm-needing-semi.stderr

```diff
@@ -28,7 +28,7 @@ LL |         false => async_dummy().await,
 help: consider removing this semicolon
    |
 LL |             async_dummy()
-   |                         --
+   |                         /\
```

Comment:
This makes me think that underline + extra space might be better than `/\`...

Reply:
Changed the output radically, please take a look at the new `diff`-like format.